assets/js/main.js

```diff
@@ -0,0 +1,308 @@
+var taskName = document.getElementById("taskName");
+var assignee = document.getElementById("assignee");
+
+var addTaskButton = document.getElementById("addTaskButton");
+var clearTaskButton = document.getElementById("clearTaskButton");
+
+var tasksData = document.getElementById("tasksData");
+var tasksDataDone = document.getElementById("tasksDataDone");
+
+var tasksArray = [];
+
+var deleteIcon = document.querySelector(".deleteIcon");
+var deleteAllButton = document.querySelector("#deleteAllButton");
+
+var currentIndex;
+
+var countTask = document.querySelector("#countTask");
+var countDone = document.querySelector("#countDone");
+
+var toggledBtn = document.querySelector("#toggledBtn");
+var toggle = 0;
+
+var nameAlert = document.getElementById("nameAlert");
+
+// to check if the local storage empty or not
+if (localStorage.getItem("tasksList") == null) {
+  tasksArray = [];
+} else {
+  tasksArray = JSON.parse(localStorage.getItem("tasksList"));
+  displayData();
+  counter();
+}
+// ----------------------------------------------------------------------------------------------------------
+// Add Task Button
+addTaskButton.onclick = function () {
+  if (addTaskButton.innerHTML == "Add-Task") {
+    addTask();
+    // sweet alert
+    Swal.fire({
+      position: "top-center",
+      icon: "success",
+      title: "The task has been added successfully",
+      showConfirmButton: false,
+      timer: 1500,
+    });
```

Comment:
Pls move this to its own function and then call it here
```suggestion
const showSuccessPopup = () => {
    Swal.fire({
      position: "top-center",
      icon: "success",
      title: "The task has been added successfully",
      showConfirmButton: false,
      timer: 1500,
    });
  }
  
  showSuccessPopup();
```

of course, you will need to create this `showSuccessPopup` function outside the onClick event.

you can also make this function more customizable by making it accept an argument called `successMessage` and then set  `title: successMessage,`. 

Now you will be able to use this function for both add and update task.